pipeline/setup/sql/02_setup.sql

```diff
@@ -60,12 +60,15 @@ CREATE TYPE category AS ENUM ('QUAY', 'ENTRANCE', 'PARKING', 'ACCESS_SPACE', 'SI
  * Create access_spaces table:
  * Table for the access spaces that will be generated from the paths.
  * This table will be filled in the "routing" step of the pipeline.
- * The ids of the nodes have to be stored as BIGINT because the OSM ids are too big for INT.
+ * The osm_ids of the nodes have to be stored as BIGINT because the OSM ids are too big for INT.
+ * The constraint PK_id is used to make sure, that there is only one access space per node and level.
  */
 CREATE TABLE access_spaces (
-  osm_id BIGINT PRIMARY KEY,
+  osm_id BIGINT NOT NULL,
   osm_type CHAR(1),
-  "IFOPT" TEXT,
+  "level" TEXT NOT NULL,
+  "IFOPT" TEXT NOT NULL,
   tags jsonb,
-  geom GEOMETRY
+  geom GEOMETRY,
+  CONSTRAINT PK_id UNIQUE (osm_id,"level")
 );
```

Comment:
Currently this table doesn't have a primary key. If now every access space has a level then I would make it like this:

`CONSTRAINT PK_id PRIMARY KEY (node_id,level),`

I would now also drop `osm_type CHAR(1),` becaue it seems unlikely to me that we will ever import access spaces in the near future (also because it would introduce a lot of difficulties)

Reply:
Implemented in [ba9dcbd](https://github.com/OPENER-next/osm2vdv462/commit/ba9dcbdd96b85776bc3d53e8c6226ba333b0de73)

Reply:
Then we could remove the creation of the `final_access_spaces` view, as it would not be used in the `stop_area_elements` view.